sql-server-2014-alwayson-existing-vnet-and-ad/azuredeploy.json

```diff
@@ -23,474 +21,165 @@
   },
-
   "resources": [
-
-    {
-      "type": "Microsoft.Compute/availabilitySets",
-      "name": "[variables('sqlAvailabilitySetName')]",
-      "apiVersion": "[variables('apiVersionNetworks')]",
-      "location": "[resourceGroup().location]"
-    },
-
-    {
-      "name": "[variables('deploySqlClusterVms')]",
-      "type": "Microsoft.Resources/deployments",
-      "apiVersion": "[variables('apiVersion')]",
-      "dependsOn": [
-        "[resourceId('Microsoft.Compute/availabilitySets', variables('sqlAvailabilitySetName'))]"
-      ],
-      "properties": {
-        "mode": "Incremental",
-        "templateLink": {
-          "uri": "[variables('sqlClusterTemplateURL')]",
-          "contentVersion": "1.0.0.0"
-        },
-        "parameters": {
-          "adminUsername": { "value": "[parameters('adminUsername')]" },
-          "adminPassword": { "value": "[parameters('adminPassword')]" },
-          "storageAccount": { "value": "[variables('SqlStorageName')]" },
-          "storageAccountType": { "value": "[parameters('storageAccountType')]" },
-          "nicSubnetUri": { "value": "[variables('sqlSubnetRef')]" },
-          "lbSubnetUri": { "value": "[variables('sqlSubnetRef')]" },
-          "vmNamePrefix": { "value": "[variables('sqlClusterVmNamePrefix')]" },
-          "sqlLBIPAddress": { "value": "[parameters('sqlLBIPAddress')]" },
-          "vmSize": { "value": "[parameters('sqlVMSize')]" },
-          "availabilitySetName": { "value": "[variables('sqlAvailabilitySetName')]" }
-        }
-      }
-    },
-
-    {
-      "name": "[variables('deploySqlWitnessVms')]",
-      "type": "Microsoft.Resources/deployments",
-      "apiVersion": "[variables('apiVersion')]",
-      "dependsOn": [
-        "[resourceId('Microsoft.Compute/availabilitySets', variables('sqlAvailabilitySetName'))]"
-      ],
-      "properties": {
-        "mode": "Incremental",
-        "templateLink": {
-          "uri": "[variables('sqlWitnessTemplateURL')]",
-          "contentVersion": "1.0.0.0"
-        },
-        "parameters": {
-          "adminUsername": { "value": "[parameters('adminUsername')]" },
-          "adminPassword": { "value": "[parameters('adminPassword')]" },
-          "storageAccount": { "value": "[variables('sqlWitnessStorageName')]" },
-          "subnetUri": { "value": "[variables('sqlSubnetRef')]" },
-          "storageAccountType": { "value": "[parameters('storageAccountType')]" },
-          "vmNamePrefix": { "value": "[variables('sqlWitnessVmNamePrefix')]" },
-          "vmSize": { "value": "[parameters('sqlWitnessVMSize')]" },
-          "availabilitySetName": { "value": "[variables('sqlAvailabilitySetName')]" }
-        }
-      }
-    },
-
     {
-      "name": "[variables('updateSQLWNic')]",
+      "name": "[variables('deploySqlCluster')]",
       "type": "Microsoft.Resources/deployments",
       "apiVersion": "[variables('apiVersion')]",
```

Comment:
Use literal strings for the apiVersions, otherwise schema validation cannot be done... (just one comment for all the instances)